arch/arm/src/nrf52/nrf52_lowputc.c

```diff
@@ -270,7 +270,7 @@ static void nrf52_setparity(uintptr_t base,
  * Name: nrf52_setstops
  ****************************************************************************/
 
-#ifdef HAVE_UART_STOPBITS
+#if defined(HAVE_UART_DEVICE) && defined(HAVE_UART_STOPBITS)
```

Comment:
Why not just wrap all of these in `HAVE_UART_DEVICE` instead of putting it around every function?  I think that how most of the other arch are done.  Then its really easy to see the couple required functions that are not conditioned on it.

Reply:
done